units/provisioning_setup_host.go

```diff
@@ -833,3 +839,53 @@ func parseLsblkOutput(lsblkOutput string) ([]blockDevice, error) {
 
 	return devices.BlockDevices, nil
 }
+
+// GetGithubTokensForTask returns a read-only token for owner/repo associated with the task
+// and a read-only token for each module associated with the task.
+func GetGithubTokensForTask(taskId string, ctx context.Context) (string, []string) {
+	// Do not error at any point while trying to populate github tokens
+	// because if the repo is not private, cloning the repo will still work.
```

Comment:
If the repo is not private, it's okay if for example, github is down and we couldn't get a token, because the clone will still work. If it's private and we failed to get a token for whatever reason, the clone will fail.